package.json

```diff
@@ -28,50 +28,51 @@
     "react-dom": ">=16.8.0 <18.0.0"
   },
   "devDependencies": {
-    "@fluentui/react": "^7.162.1",
-    "@types/enzyme": "^3.1.15",
-    "@types/enzyme-adapter-react-16": "^1.0.3",
-    "@types/jest": "^26.0.20",
-    "@types/react": "^16.7.14",
-    "@types/react-dom": "^16.0.11",
+    "@fluentui/react": "^7.204.0",
+    "@types/enzyme": "^3.10.12",
+    "@types/enzyme-adapter-react-16": "^1.0.6",
+    "@types/jest": "^26.0.24",
+    "@types/react": "^16.14.35",
+    "@types/react-dom": "^16.9.18",
     "babel-core": "^6.26.3",
     "babel-jest": "^26.6.3",
-    "babel-loader": "^8.0.4",
+    "babel-loader": "^8.3.0",
     "babel-plugin-named-asset-import": "^0.2.3",
-    "babel-plugin-transform-amd-to-commonjs": "^1.4.0",
+    "babel-plugin-transform-amd-to-commonjs": "^1.6.0",
     "babel-preset-react-app": "^6.1.0",
-    "bfj": "^6.1.1",
+    "bfj": "^6.1.2",
     "chalk": "^2.4.1",
-    "css-loader": "^2.0.0",
-    "enzyme": "^3.8.0",
-    "enzyme-adapter-react-16": "^1.7.1",
-    "enzyme-to-json": "^3.3.5",
+    "css-loader": "^2.1.1",
+    "enzyme": "^3.11.0",
+    "enzyme-adapter-react-16": "^1.15.7",
+    "enzyme-to-json": "^3.6.2",
+    "fetch-mock": "^9.11.0",
     "fs": "0.0.1-security",
     "fs-extra": "^7.0.1",
     "identity-obj-proxy": "^3.0.0",
     "jest": "^26.6.3",
     "jest-fetch-mock": "^2.1.2",
     "jest-junit-reporter": "^1.1.0",
-    "jest-pnp-resolver": "^1.0.2",
+    "jest-pnp-resolver": "^1.2.3",
     "node-sass": "^4.12.0",
     "path": "^0.12.7",
-    "postcss-flexbugs-fixes": "^4.1.0",
+    "postcss-flexbugs-fixes": "^4.2.1",
     "postcss-loader": "^3.0.0",
-    "postcss-preset-env": "^6.5.0",
+    "postcss-preset-env": "^6.7.1",
     "react": "^16.14.0",
     "react-app-polyfill": "^0.1.3",
     "react-dev-utils": "^11.0.3",
     "react-dom": "^16.14.0",
-    "sass-loader": "^7.1.0",
+    "sass-loader": "^7.3.1",
     "style-loader": "^0.23.1",
     "terser-webpack-plugin": "^1.1.0",
-    "ts-jest": "^26.5.2",
-    "ts-loader": "^5.3.1",
-    "tslint": "^5.11.0",
-    "tslint-microsoft-contrib": "^6.0.0",
+    "ts-jest": "^26.5.6",
+    "ts-loader": "^5.4.5",
+    "tslint": "^5.20.1",
+    "tslint-microsoft-contrib": "^6.2.0",
     "tslint-react": "^3.6.0",
-    "typescript": "^3.2.2",
-    "webpack": "^4.27.1"
+    "typescript": "^3.9.10",
```

Comment:
TypeScript doesn't follow semantic versioning, so even though npm updated this to the latest minor version, you can really update it all the way to 4.9 if you want.